ubatch/ubatch.py

```diff
@@ -142,15 +142,15 @@ def _handle_request(self) -> None:  # pragma: no cover
         while not self._stop_thread:
             self._procces_in_batch()
 
-    def ubatch(self, data: T) -> S:
+    def ubatch(self, *args: tuple, **kwargs: dict) -> S:
```

Comment:
I think here we breake backward compability, developers using `foo.ubatch(data="bar")` are going to have problems